plugins/inputs/aerospike/README.md

```diff
@@ -56,21 +55,33 @@ are available from the aerospike `namespace/<namespace_name>` command.
+aerospike_latency,aerospike_host=localhost:3000,cluster=cache,host=Aerospike-Cache,namespace=test,node_name=BB93B8365DD8E06,service=telegraf latency_read-ops=24358,latency_read>1ms=2.4358,latency_read>8ms=0,latency_read>64ms=0 1538725061000000000
```

Comment:
I think we don't need to prefix the fields with latency because the measurement name includes it.  We should also use underscore instead of hyphen for consistency and `gt` instead of `>` to avoid escaping and reserve character use:

`latency_read-ops` -> `read_ops`
`latency_read>1ms` -> `read_gt_8ms`

Reply:
Correct, I'll remove the latency and use underscore everywhere for a uniqueness of metrics.